### ISSUE TITLE
On drag to slide long last slides show blank for few millisecond

### ISSUE THREAD

**Issue by Jaimindarji07** [kenwheeler/slick#3764]
Here is my JSFiddle link - https://jsfiddle.net/jaimin07/2ug9fdqb/15/

Try to drag from the last slide to 3-4 slide some times slide appear after some millisecond.

**Comment by ahmadalfy** [comment]
Yes because the infinite option waits until the dragging is released then rearrange the elements and add new elements near the end. I am not sure how this can be fixed.
